PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+<!-- 
+Thank you for submitting a pull request! Please add some info (if applicable) to give us some context on the PR.
+
+We will review the PR as soon as possible, leave feedback, add a tag, etc. and let you know what's going on.
+
+Cheers!
+
+The App Center team -->
+
+Things to consider before you submit the PR:
```

Comment:
Is it useful to add something about the test apps like changes should have been tested in the test apps and test apps should be updated accordingly ?